daemon/api_test.go

```diff
@@ -104,7 +104,22 @@ func (s *apiSuite) SetUpTest(c *check.C) {
 	s.overlord = &fakeOverlord{
 		configs: map[string]string{},
 	}
-	s.daemon = newTestDaemon(c)
+	s.d = nil
+}
+
+func (s *apiSuite) TearDownTest(c *check.C) {
+	s.d = nil
+}
+
+func (s *apiSuite) daemon(c *check.C) *Daemon {
+	if s.d != nil {
```

Comment:
why store it on s and panice if we call it again, given that we don't use s.d anymore ?